src/querylogger.py

```diff
@@ -0,0 +1,41 @@
+import logging

```

Comment:
class names like this should be `QueryLogger.py`

Reply:
Solved. @josephmancuso There some way to import some attributes from databases.py into some files located in masoniteorm folder?